src/state/routing/slice.ts

```diff
@@ -56,14 +64,15 @@ export const routing = createApi({
 
               // NO_ROUTE should be treated as a valid response to prevent retries.
               if (typeof data === 'object' && data.errorCode === 'NO_ROUTE') {
-                return { data: NO_ROUTE as GetQuoteResult }
+                return { data: NO_ROUTE as TradeQuoteResult }
```

Comment:
so that function definition is a very complicated type using `QueryReturnValue` that rtk-query does not allow you to export 🙃 

but the return type is already inferred from us setting the type of `baseQuery` [here](https://github.com/Uniswap/widgets/blob/c4c49155081bbe6cbb63a92d802cc59da742267f/src/state/routing/slice.ts#L19)

i had to cast it like i had previously because the rtk-query typechecker didn't like that `TradeQuoteResult` is technically a union, and returning a subtype is seen as not valid